pkg/controller/import-controller.go

```diff
@@ -425,13 +434,42 @@ func (r *ImportReconciler) updatePvcFromPod(pvc *corev1.PersistentVolumeClaim, p
 			r.recorder.Event(pvc, corev1.EventTypeNormal, ImportSucceededPVC, "Import Successful")
 			log.V(1).Info("Completed successfully, deleting POD", "pod.Name", pod.Name)
 		}
+		if getSource(pvc) == SourceVDDK {
+			r.saveVddkAnnotations(pvc, pod, log)
+		}
 		if err := r.client.Delete(context.TODO(), pod); IgnoreNotFound(err) != nil {
 			return err
 		}
 	}
 	return nil
 }
 
+func (r *ImportReconciler) saveVddkAnnotations(pvc *corev1.PersistentVolumeClaim, pod *corev1.Pod, log logr.Logger) {
```

Comment:
Done.

Reply:
I undid this and removed the Update inside saveVddkAnnotations in favor of moving it higher up in updatePvcFromPod, so it can piggyback on an existing Update. Before this, I noticed I was causing some errors about modifying stale objects.